Makefile

```diff
@@ -16,6 +16,7 @@ EXTLDFLAGS := -static-libstdc++ -static-libgcc
 endif
 
 GOTAGS          := --tags "sqlite_unlock_notify sqlite_omit_load_extension"
+GOTRIMPATH	:= $(shell go help build | grep -q .-trimpath && echo -trimpath)
```

Comment:
it's an interesting way to detect if `trimpath` is supported by the current go compiler..